libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/TelemetryLocationAndProgressDispatcher.kt

```diff
@@ -33,7 +33,12 @@ internal class TelemetryLocationAndProgressDispatcher(scope: CoroutineScope) :
         AtomicReference(
             RouteProgressWithTimestamp(
                 0,
-                RouteProgress.Builder(DirectionsRoute.builder().build()).build()
+                RouteProgress.Builder(
+                    DirectionsRoute.builder()
```

Comment:
`distance` and `duration` [mustn't be null ](https://github.com/mapbox/mapbox-java/blob/aa126fcad9e7dba2f5b7f74c5a46291463d9b85e/services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/DirectionsRoute.java#L50-L60)